templates/controllers/grid/catalogEntry/editFormat.tpl

```diff
@@ -15,8 +15,20 @@
 </script>
 <div id="editPublicationFormatMetadataTabs">
 	<ul>
-		<li><a href="{url router=PKPApplication::ROUTE_COMPONENT op="editFormatTab" submissionId=$submissionId representationId=$representationId publicationId=$publicationId}">{translate key="common.edit"}</a></li>
-		<li><a href="{url router=PKPApplication::ROUTE_COMPONENT op="editFormatMetadata" submissionId=$submissionId representationId=$representationId publicationId=$publicationId}">{translate key="submission.informationCenter.metadata"}</a></li>
+		<li>
+			<a href="{url router=PKPApplication::ROUTE_COMPONENT op="editFormatTab" submissionId=$submissionId representationId=$representationId publicationId=$publicationId}">
+				{translate key="common.edit"}
+			</a>
+		</li>
+
+		{if !isset($addFormat)}
```

Comment:
I think rather than introducing a `$addFormat` template variable, you can just check `$representationId` to see if it contains anything.

Reply:
good idea, updated .